nikolai4D/ContechOS
# C2 6 link title should be set automatically for type data rels and instance data rels

## Description & motivation



- When creating a rel between two nodes (all except configDef) and a parentRel has been picked -> Title input get automatically filled with its title

## How to test



- Click on a node
- Click "Link"
- Click on another node
- Pick a parentRel from dropDown
- See change in Title

## New tickets to be added



None

---
